tests/tck/features/optimizer/PrunePropertiesRule.feature

```diff
@@ -391,6 +392,24 @@ Feature: Prune Properties rule
   @distonly
   Scenario: return function
     Given a graph with space named "nba"
+    When profiling query:
+      """
+      MATCH (v1)-[:like]->(v2)
+      WHERE id(v1) == "Tim Duncan"
+      RETURN count(v2), v1
+      """
+    Then the result should be, in order:
+      | count(v2) | v1                                                                                                          |
+      | 2         | ("Tim Duncan" :bachelor{name: "Tim Duncan", speciality: "psychology"} :player{age: 42, name: "Tim Duncan"}) |
+    And the execution plan should be:
+      | id | name           | dependencies | operator info                                                                                                                                                                                                                                                             |
+      | 7  | Aggregate      | 6            |                                                                                                                                                                                                                                                                           |
+      | 6  | Project        | 5            |                                                                                                                                                                                                                                                                           |
+      | 5  | AppendVertices | 4            | {  "props": "[{\"props\":[\"_tag\"],\"tagId\": 3}, {\"props\":[\"_tag\"],\"tagId\": 4}, {\"props\":[\"_tag\"],\"tagId\": 5}]" }                                                                                                                                           |
```

Comment:
better to remove the tagId field in plan since it depends on runtime context.

Reply:
ok